packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-tree-item.tsx

```diff
@@ -83,6 +84,18 @@ export const PageTreeItem = forwardRef<HTMLAnchorElement, PageTreeItemProps>(
     return collapsed && isSorting ? null : (
       <Box
         ref={wrapperRef}
+        onContextMenu={(event) => {
+          if (popupState.isOpen) {
+            setAnchorPosition(undefined);
+            return popupState.close();
+          }
+          event.preventDefault();
+          setAnchorPosition({
+            left: event.clientX + 2,
+            top: event.clientY - 6,
+          });
+          popupState.open(event);
+        }}
```

Comment:
I think it would be better to move `event.preventDefault()` to the top to prevent the default menu from opening when right clicking with the page menu open. What do you think?

![menu2](https://user-images.githubusercontent.com/37453800/193059570-caa9b0bc-553a-414e-865d-ea7db5e8c6b1.gif)

```suggestion
        onContextMenu={(event) => {
          event.preventDefault();
          if (popupState.isOpen) {
            setAnchorPosition(undefined);
            return popupState.close();
          }
          setAnchorPosition({
            left: event.clientX + 2,
            top: event.clientY - 6,
          });
          popupState.open(event);
        }}
```